android/res/values/strings.xml

```diff
@@ -104,6 +104,8 @@
   <string name="preferences_disable_barcode_scene_mode_title">No barcode scene mode</string>
   <string name="preferences_disable_continuous_focus_summary">Use only standard focus mode</string>
   <string name="preferences_disable_continuous_focus_title">No continuous focus</string>
+  <string name="preferences_disable_history">Barcode history is not automatically saved</string>
```

Comment:
Just this single string or both strings that reference history? And, where can I find the tool?